pretty_gpx/common/data/overpass_request.py

```diff
@@ -1,31 +1,198 @@
 #!/usr/bin/python3
 """Overpass API."""
 
-import overpy
+import urllib.parse
+import urllib.request
+from dataclasses import dataclass
+
+import ujson
+from overpy import Area
+from overpy import Node
+from overpy import Overpass
+from overpy import Relation
+from overpy import Result
+from overpy import Way
 
 from pretty_gpx.common.gpx.gpx_bounds import GpxBounds
 from pretty_gpx.common.gpx.gpx_track import GpxTrack
 from pretty_gpx.common.utils.logger import logger
 from pretty_gpx.common.utils.profile import profile
+from pretty_gpx.common.utils.profile import Profiling
 
 DEBUG_OVERPASS_QUERY = False
 
 
 ListLonLat = list[tuple[float, float]]
 
 
+@dataclass
+class OverpassQuery:
+    """Class to store all queries needed, to launch them and to split the result."""
+    query_dict: dict[str,str]
+    query_unprocessed_results: dict[str,Result]
+    query_cached_results: dict[str,str]
+
+    def __init__(self) -> None:
```

Comment:
skip this init and add default empty fields with ` = field(default_factory=dict)` ?